src/hotspot/cpu/x86/macroAssembler_x86.hpp

```diff
@@ -1489,8 +1489,14 @@ class MacroAssembler: public Assembler {
   void vpxor(XMMRegister dst, XMMRegister nds, AddressLiteral src, int vector_len, Register scratch_reg = rscratch1);
 
   // Simple version for AVX2 256bit vectors
-  void vpxor(XMMRegister dst, XMMRegister src) { Assembler::vpxor(dst, dst, src, true); }
-  void vpxor(XMMRegister dst, Address src) { Assembler::vpxor(dst, dst, src, true); }
+  void vpxor(XMMRegister dst, XMMRegister src) {
+    assert(UseAVX >= 2, "Only with AVX2");
```

Comment:
I would suggest the assert msg as "Only with AVX2 or above".
We should also update the copyright year.

Reply:
Rephrased the assert message, updated copyright date. Thanks!